### ISSUE TITLE
[Screen reader - WIX - User Cancelled installation] Narrator doesn't announce the name of title in scan mode under 'User Cancelled installation' pop-up window.

### ISSUE THREAD

**Issue by tgugler** [wixtoolset/issues#6228]
If this issue is a bug:

* Which version of WiX are you building with?

> 3.14.0.1119

* Which version of Visual Studio are you building with (if any)?

> Visual Studio 2017

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> 1.0.0.4

* Which version of .NET are you building with?

> Not Applicable

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Not Applicable

* Describe the problem and the steps to reproduce it.

User Experience:
User who depends upon screen reader will not get the complete information of pop-up window if narrator is not reading the title name of 'User Cancelled installation' pop-up window.

Pre Requisite:
Download exe file from https://www.microsoft.com/en-us/download/details.aspx?id=54688 ​
​
Repro Steps:​
1. Launch the "NPS Extension For Azure MFA" setup file.
2. Navigate to "MICROSOFT SOFTWARE LICENSE TERMS" popup window and select "Install" button.​
3. After successful installation of Setup, again install the same setup.​
4. Navigate to "Modify Setup" popup window and select "repair" button.​
5. Navigate to "Setup Progress" popup window and select "cancel" button.​
6. Navigate all the controls of "User Cancelled installation" popup.​
7. Verify that narrator announces the name of title in scan mode or not.

Actual Result:
Narrator is not reading the name of title while navigating in scan mode under 'User Cancelled installation' pop-up window. It is reading only 'title bar', due to this user will not know about the name of pop-up window.

Expected Result:
Narrator should be provide the name of title while navigating in scan mode under 'User Cancelled installation' pop-up window. It should be read as 'NPS extension for azure MFA setup' title bar.

Note:
1. This issue does not exists with another screen reader (JAWS and NVDA).
2. Same issue exists under 'Setup failed' pop-up.

[MAS4.2.1_User Cancelled installation_Title name in scan mode.wmv.zip](https://github.com/wixtoolset/issues/files/5043444/MAS4.2.1_User.Cancelled.installation_Title.name.in.scan.mode.wmv.zip)
